### PR TITLE
Update docs for MAUI and WinUI

### PR DESCRIPTION
**Pull request by nCastle1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,55 +1,51 @@
...
 [![Link: NuGet](https://img.shields.io/badge/NuGet-633b9b?style=flat-square&logo=nuget&labelColor=gray)](https://www.nuget.org/profiles/Esri_Inc)
-[![Link: Esri Community](https://img.shields.io/badge/🙋-Get%20help%20in%20Esri%20Community-633b9b?style=flat-square)](https://community.esri.com/t5/arcgis-runtime-sdk-for-net/bd-p/arcgis-runtime-sdk-dotnet-questions)
+[![Link: Early Adopter Community](https://img.shields.io/badge/🙋-Get%20help%20in%20Early%20Adopter%20Community-633b9b?style=flat-square)](https://esriurl.com/agsrt200beta)
 
-# ArcGIS Runtime SDK for .NET - Toolkit
+# ArcGIS Runtime SDK for .NET - Toolkit 200 beta
 
 The ArcGIS Runtime SDK for .NET Toolkit contains controls and components to accelerate your development with [ArcGIS Runtime SDK for .NET](https://developers.arcgis.com/net/).
 
+> **IMPORTANT** This is a beta branch. The 200.0.0-beta release of ArcGIS Runtime is beta software. If you would like to use this software, please join the [Esri Early Adopter community](https://esriurl.com/agsrt200beta).
```
Comment (pMaske) [inline README.md:12]:
```suggestion
> **IMPORTANT** This is branch is under 200.0.0-beta program and showcases changes we will be releasing in our upcoming 200.0.0. Please join the [Esri Early Adopter community](https://esriurl.com/agsrt200beta) to provide feedback or report issues.
```

File: README.md [hunk 2]
```diff
@@ -1,55 +1,51 @@
...
-![Badge: Toolkit](https://img.shields.io/badge/Toolkit-black?style=flat-square)
-[![Link: Templates](https://img.shields.io/badge/Templates-633b9b?style=flat-square&logo=visualstudio&labelColor=gray)](https://github.com/Esri/arcgis-runtime-templates-dotnet)
+[![Link: Templates](https://img.shields.io/badge/Templates-633b9b?style=flat-square&logo=visualstudio&labelColor=gray)](https://www.nuget.org/packages?q=ArcGIS+Runtime+Templates)
 [![Link: NuGet](https://img.shields.io/badge/NuGet-633b9b?style=flat-square&logo=nuget&labelColor=gray)](https://www.nuget.org/profiles/Esri_Inc)
-[![Link: Esri Community](https://img.shields.io/badge/🙋-Get%20help%20in%20Esri%20Community-633b9b?style=flat-square)](https://community.esri.com/t5/arcgis-runtime-sdk-for-net/bd-p/arcgis-runtime-sdk-dotnet-questions)
+[![Link: Early Adopter Community](https://img.shields.io/badge/🙋-Get%20help%20in%20Early%20Adopter%20Community-633b9b?style=flat-square)](https://esriurl.com/agsrt200beta)
 
-# ArcGIS Runtime SDK for .NET - Toolkit
+# ArcGIS Runtime SDK for .NET - Toolkit 200 beta
```
Comment (pMaske) [inline README.md:8]:
```suggestion
# ArcGIS Runtime SDK for .NET - Toolkit 200.0.0- beta release
```

**Review by pMaske — APPROVED** [review]
Thanks @nCastle1. Just two minor updates otherwise looks good.

File: README.md (hunk 2 above)
Comment (dotMorten) [reply README.md:8]: I don't think we should explicitly state "beta" here or anywhere else. This is perpetually the v200 branch.

File: README.md [hunk 3]
```diff
@@ -1,55 +1,51 @@
...
-[![Link: Toolkit NuGet - AR](https://img.shields.io/badge/Augmented%20Reality%20Toolkit-007ac2?style=flat-square&labelColor=gray&logo=nuget)](https://www.nuget.org/packages/Esri.ArcGISRuntime.ARToolkit/)
-[![Link: Toolkit NuGet - Forms AR](https://img.shields.io/badge/Xamarin.Forms%20Augmented%20Reality%20Toolkit-007ac2?style=flat-square&labelColor=gray&logo=nuget)](https://www.nuget.org/packages/Esri.ArcGISRuntime.ARToolkit.Forms/)
+[![Link: Toolkit NuGet - WPF](https://img.shields.io/badge/WPF-007ac2?style=flat-square&labelColor=gray&logo=windowsxp)](https://www.nuget.org/packages/Esri.ArcGISRuntime.Toolkit.WPF/)
+[![Link: Toolkit NuGet - WinUI](https://img.shields.io/badge/WinUI-007ac2?style=flat-square&labelColor=gray&logo=windows11)](https://www.nuget.org/packages/Esri.ArcGISRuntime.Toolkit.WinUI/)
+[![Link: Toolkit NuGet - UWP](https://img.shields.io/badge/UWP-007ac2?style=flat-square&labelColor=gray&logo=windows)](https://www.nuget.org/packages/Esri.ArcGISRuntime.Toolkit.UWP/)
+[![Link: Toolkit NuGet - MAUI](https://img.shields.io/badge/MAUI-007ac2?style=flat-square&labelColor=gray&logo=dotnet)](https://www.nuget.org/packages/Esri.ArcGISRuntime.Toolkit.Maui/)
```
Comment (dotMorten) [inline README.md:48]: These are links to nuget packages, not platforms. I don't get why these badges are changed?

**Review by dotMorten — APPROVED** [review]
I would remove any explicit statement of beta. This is the v200 branch, not the v200-beta branch.

**Review by dotMorten — APPROVED** [review]

**Review by pMaske — APPROVED** [review]

**Pull request merged**